modules/aioseop_importer_exporter.php

```diff
@@ -146,18 +146,18 @@ function debug_post_types() {
 					$this->default_options['import_export_help']['default'] .= "\n</ul>\n";
 				} else {
 					$this->default_options['import_export_help']['default'] .= '<br />'
-																			   . __(
-																				   'There are no other modules currently loaded!',
-																				   'all-in-one-seo-pack'
-																			   );
+																			. __(
```

Comment:
White space needs to be removed, and could probably just remove the new lines.

Reply:
thats what it did.

Reply:
Whoa...that is one hell of an indent. I didn't even see the text til now.